packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx

```diff
@@ -34,9 +34,21 @@ describe('HeaderLockup', () => {
     expect(headerMenu).toBeInTheDocument();
     expect(headerMenu).toHaveAttribute('title', 'Site navigation menu');
 
+    expect(screen.getByRole('link', { name: 'Help' })).toHaveAttribute(
+      'href',
+      'https://support.mozilla.org/products/mozilla-account'
```

Comment:
Do we want/need any UTM params for SUMO?

Reply:
There weren't any listed in the ticket, so I don't think so. We'll get metrics info on our side from the new Glean event added here 🤷‍♀️